src/main/steps/applicant1/interim-applications/bailiff-service/partner-date-of-birth/content.ts

```diff
@@ -0,0 +1,150 @@
+import { CaseDate } from '../../../../../app/case/case';
+import { YesOrNo } from '../../../../../app/case/definition';
+import { TranslationFn } from '../../../../../app/controller/GetController';
+import { FormContent, FormFieldsFn } from '../../../../../app/form/Form';
+import { covertToDateObject } from '../../../../../app/form/parser';
+import {
+  areDateFieldsFilledIn,
+  isDateInputInvalid,
+  isFieldFilledIn,
+  isFutureDate,
+  isValidNumber,
+} from '../../../../../app/form/validation';
+import { SupportedLanguages } from '../../../../../modules/i18n';
+
+const en = () => ({
+  title: 'Your partner',
```

Comment:
I am not too sure if `partner` needs to be conditional just like we have on other pages to get either husband or wife. It is on all the subsequent pages as if we need to refer to husband or wife 🤔